asammdf/gui/ui/file_widget.ui

```diff
@@ -172,7 +172,7 @@
             <string>Convert</string>
            </property>
            <property name="icon">
-            <iconset resource="resource.qrc">
+            <iconset resource="../../../../../asammdf/asammdf/gui/ui/resource.qrc">
```

Comment:
Hey @freakatzz,

it is necessary to remove the relative path for occurrences (also line 186, 220, 375, and so on)